contracts/utils/CAIP10.sol

```diff
@@ -13,6 +13,12 @@ import {CAIP2} from "./CAIP2.sol";
  * account_id:        chain_id + ":" + account_address
  * chain_id:          [-a-z0-9]{3,8}:[-_a-zA-Z0-9]{1,32} (See {CAIP2})
  * account_address:   [-.%a-zA-Z0-9]{1,128}
+ *
+ * WARNING: According to [CAIP-10's canonicalization section](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md#canonicalization),
+ * the specification remains at the developer's discretion. However, case sensitivity or other transformations
```

Comment:
"the specification" of what?

I'm not sure However is the right world to use here ... or maybe I'm missunderstanding.